src/test/java/org/jenkinsci/plugins/envinject/BuildCauseRetrieverTest.java

```diff
@@ -1,24 +1,24 @@
 package org.jenkinsci.plugins.envinject;
 
+import static com.google.common.base.Joiner.*;
```

Comment:
please dont use wildcard imports. They hamper to see which classes is actually imported